### PR TITLE
[Bug fix] Microsite verification fix

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Users would get stuck in an email verification loop if they were verified with a different account than they were logged in as, this logs them out in that case and sends them to the login screen to prevent issues.

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: (PUT IT HERE)
- Process.st launch checklist: (PUT IT HERE)

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

**Comment by 00salmon** [comment]
there's some conflicts from main atm that need addressing but otherwise should be ready to 🚀

File: packages/mint-components/CHANGELOG.md [hunk 1]
```diff
@@ -954,7 +968,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - \<sqm-popup-container>
   - \<sqm-stencilbook>
 
-[unreleased]: https://github.com/saasquatch/program-tools/compare/mint-components@1.8.4...HEAD
+[unreleased]: https://github.com/saasquatch/program-tools/compare/mint-components@1.8.5...HEAD
+[1.8.5]: https://github.com/saasquatch/program-tools/releases/tag/%40saasquatch%2Fmint-components%401.8.5
```
Comment (00salmon) [inline packages/mint-components/CHANGELOG.md:972]:
```suggestion
[unreleased]: https://github.com/saasquatch/program-tools/compare/mint-components@1.9.1...HEAD
[1.9.1]: https://github.com/saasquatch/program-tools/releases/tag/%40saasquatch%2Fmint-components%401.8.5
```

Comment (00salmon) [reply packages/mint-components/CHANGELOG.md:972]: also looks like `1.9.0` is missing from this list

**Review by 00salmon — COMMENTED** [review]
just a couple comments on the mint-component changelog 🙏

**Review by Locrian24 — APPROVED** [review]

**Review by 00salmon — COMMENTED** [review]
looks good to me 👍

**Review by Locrian24 — DISMISSED** [review]

**Review by Locrian24 — APPROVED** [review]

**Pull request merged**